docs/current_docs/api/constructors.mdx

```diff
@@ -163,8 +182,6 @@ When assigning default values to complex types in TypeScript, it is necessary to
 </TabItem>
 </Tabs>
 
-It is necessary to explicitly declare the type even when a default value is assigned, so that the Dagger SDK can extend the GraphQL schema correctly.
-
 Here is an example call for this Dagger Function:
```

Comment:
This part in the end is way too far from the code it applies to, which is the first code snippet in the "Default values for complex types" section.

I understand it's because of the general vs language specific parts but ends up being confusing. Maybe put the extra information inside a "tip" admonition so it's self contained?

Reply:
Switched to a partial for the output, which makes it easier to handle.